rust/operator-binary/src/format/well_known.rs

```diff
@@ -46,24 +46,24 @@ pub enum WellKnownSecretData {
 }
 
 impl WellKnownSecretData {
-    pub fn into_files(self) -> SecretFiles {
+    pub fn into_files(self, names: &NamingOptions) -> SecretFiles {
         match self {
             WellKnownSecretData::TlsPem(TlsPem {
                 certificate_pem,
                 key_pem,
                 ca_pem,
             }) => [
-                (FILE_PEM_CERT_CERT.to_string(), certificate_pem),
-                (FILE_PEM_CERT_KEY.to_string(), key_pem),
-                (FILE_PEM_CERT_CA.to_string(), ca_pem),
+                (names.tls_pem_cert_name.to_string(), certificate_pem),
+                (names.tls_pem_key_name.to_string(), key_pem),
+                (names.tls_pem_ca_name.to_string(), ca_pem),
             ]
             .into(),
             WellKnownSecretData::TlsPkcs12(TlsPkcs12 {
                 keystore,
                 truststore,
             }) => [
-                (FILE_PKCS12_CERT_KEYSTORE.to_string(), keystore),
-                (FILE_PKCS12_CERT_TRUSTSTORE.to_string(), truststore),
+                (names.tls_pkcs12_keystore_name.to_string(), keystore),
+                (names.tls_pkcs12_truststore_name.to_string(), truststore),
```

Comment:
We can avoid this cloning if we move the `NamingOptions` instead - we shouldn't need it after this anwyay.

Reply:
Done in 83fea67